custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -105,12 +101,19 @@ def monitor(self, kwargs, predictions, predict_time_ms):
 
     def predict(self, **kwargs):
         start_predict = time.time()
-        predictions, labels = self._predict(**kwargs)
+        predictions, labels_in_predictions = self._predict(**kwargs)
+        labels_in_request = (
+            None
+            if self._target_type in {TargetType.REGRESSION, TargetType.ANOMALY}
+            else get_request_labels(
+                self._class_labels, self._positive_class_label, self._negative_class_label,
+            )
```

Comment:
nit: would be better to reverse the if statement to use get_request_labels if in CLASSIFICATION, MULTICLASS, in case we ever add a new target type

Reply:
updated
https://github.com/datarobot/datarobot-user-models/blob/dbb132a4219adad47006818838547a10597254ce/custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py#L105-L111

https://github.com/datarobot/datarobot-user-models/blob/dbb132a4219adad47006818838547a10597254ce/custom_model_runner/datarobot_drum/drum/model_adapter.py#L449-L455